code/languages/Flint/models/Flint.migration.mps

```diff
@@ -21,8 +21,10 @@
     <import index="87nw" ref="r:ca2ab6bb-f6e7-4c0f-a88c-b78b9b31fff3(de.slisson.mps.richtext.structure)" />
```

Comment:
Do something like this in a util method instead of using deprecated setProperty in SplitActiveLanguage
![image](https://user-images.githubusercontent.com/15971736/179503680-80d71b34-d4e7-4f2b-aba4-19ec50ecd5e5.png)

Reply:
Implemented feedback with `concept/EditorLanguage/.getProperties().spliterator()` since `editorlang/.getProperties` retuned empty. "Offically" we should not use names as keys, but MPS also admits its still the case and they haven't migrated.

![image](https://user-images.githubusercontent.com/37440544/179520517-a0cb5e6f-c43e-4888-8260-ff8765deaf1f.png)